modules/ejbca-ejb-cli/src/org/ejbca/ui/cli/ca/CaImportCertDirCommand.java

```diff
@@ -321,4 +390,11 @@ public String getFullHelpText() {
     protected Logger getLogger() {
         return log;
     }
+    
+    private Certificate loadCertificateFromFile(final String filename) throws IOException, CertificateParsingException {
```

Comment:
There exists a helper class to do this. CertTools.getCertsFromPEM((String certFilename, Class<T> returnType)). Use the helper class instead.

Reply:
OK